bindings/nodejs/lib/wallet/index.ts

```diff
@@ -5,3 +5,5 @@ export * from './account';
 export * from './wallet';
 export * from './wallet-method-handler';
 export * from '../types/wallet';
+import { migrateDbChrysalisToStardust } from '../bindings';
+export { migrateDbChrysalisToStardust };
```

Comment:
Just curious, can't this be a single export from ?

Reply:
Ah yeah probably can, will try